man/bootctl.xml

```diff
@@ -153,6 +153,21 @@
         disables the timeout while always showing the menu. When an empty string ("") is specified the
         bootloader will revert to its default menu timeout.</para></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><option>unlink</option> <replaceable>ID</replaceable></term>
+
+        <listitem><para>Removes a boot loader entry including the files it refers to. Takes a single boot
+        loader entry ID string or a glob pattern as argument. Referenced files such as kernel or initrd are
+        only removed if no other entry refers to them.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><option>cleanup</option></term>
+
+        <listitem><para>Removes files from the ESP and XBOOTLDR partitions that are not
+        referenced in any boot loader entries.</para></listitem>
```

Comment:
I think we should be more precise here. THis makes it sounds as if we would empty the whole damn ESP. What's key though is that we only remove stuff below the entry token as configured via --entry-token= (or via its default automatic value, whatever that might be). People might be too frightened to use this because it sounds overly destructive otherwise.

Also mention `--dry` here. In fact it currently has no docs at all